services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java

```diff
@@ -183,6 +188,10 @@ private void testGets(final String streamName, final Shard shard) {
 
             records = result.records();
             if (records.size() > 0) {
+                long arrivalTime = records.get(0).approximateArrivalTimestamp().toEpochMilli();
+                Long delta = Math.abs(Instant.now().minusMillis(arrivalTime).toEpochMilli());
+                // Assert that the arrival date is within a minute of the current date to make sure it unmarshalled correctly.
```

Comment:
Can we add a test to disable CBOR and make sure it also works?

Reply:
At that point it's just a normal JSON service and I think we have that covered pretty well in other service integ tests and the protocol tests.

Reply:
Yeah, fair enough